.github/workflows/action-ros-ci-with-setup-ros.yml

```diff
@@ -0,0 +1,24 @@
+name: Build and test(actions-ros + setup-ros on ubuntu:focal)
+
+on: pull_request
+
+jobs:
+  build-and-test:
+    runs-on: ubuntu-latest
+    container: ubuntu:focal
+
+    steps:
+      - name: Check out repo
+        uses: actions/checkout@v2
+
+      - name: setup ROS environment
```

Comment:
nits: It would be better to begin with upper case (`Setup`)

Reply:
Thanks! Fixed e391c38